cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
On this one, I agree with @BenWhitehead. We should run truncate as a separate job type in order to have the code broken down into clearer modules. It is harder to read when jobs combined. Also, breaking down code allows for easier error handling and the ability to see where the error has occurred.